src/stdatamodels/jwst/datamodels/schemas/specpsf.schema.yaml

```
@@ -0,0 +1,48 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/specpsf.schema"
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_exptype.schema
+- $ref: keyword_readpatt.schema
+- $ref: keyword_psubarray.schema
+- $ref: keyword_filter.schema
```

```suggestion
- $ref: keyword_readpatt.schema
- $ref: keyword_pexptype.schema
- $ref: keyword_psubarray.schema
- $ref: keyword_filter.schema
- $ref: keyword_pfilter.schema
```
Do the other keywords need the corresponding `P_` keywords (like `psubarray` has already)?

---

ok. Since the only reference file we have now is MIRI LRS fix slit I think we can leave them out. NIRSpec might provide a spec psf reference file in the future so me might need to revisit this then.